Week-1/index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>CattyBoard</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style>
+      body {
+        font-family: "Lato", sans-serif;
+      }
+
+      .sidebar {
+        height: 80%;
+        width: 50px;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: whitesmoke;
+        overflow-x: hidden;
+        padding-top: 16px;
+      }
+
+      .sidebar i {
+        padding: 6px 8px 50px 16px;
+        text-decoration: none;
+        font-size: 20px;
+        color: black;
+        display: block;
+      }
+
+      .head {
+        margin-left: 50px; /* Same as the width of the sidenav */
+        height: 30%;
```

Comment:
applying heights in percentage will be based on the breakpoint so please don't use when not required